### DISCUSSION TITLE
How to secure Coolify's frontend after installation

### DISCUSSION THREAD

**Discussion by appreciated** [coollabsio/coolify#1802] [General]
Hello Coolify Team and Community,

I'm new to Coolify, having just set it up about a week ago. The ride was pretty smooth. During this period, I've noticed that accessing services on port 9000 involves sending user credentials without encryption. This approach seems reasonable during the initial setup phase when a Traefik proxy isn't yet operational. However, given the critical nature of usernames and passwords, this should only be a temporary solution.  

I have searched the official documentation and other issues, but I could not find information, on how to resolve my issue. I propose sharing an official post installation stop to secure the Coolify service using HTTPS? It would be immensely helpful for users if guidance on securing their Coolify instances were provided in the documentation.

Many thanks for your support and for creating such an outstanding tool!

**Comment by kazzkiq** [comment]
Coolify already supports HTTPs and custom domains. Here are the steps:

1. Open your DNS provider (e.g. CloudFlare, Godaddy, Namecheap, etc) and create an A record with a subdomain of your taste pointing to your server IP. e.g. `coolify.mydomain.com`
2. Now wait about 2 minutes, log into your Coolify panel, and go to **Settings**.
3. In the field **"Instance's Domain"** type the subdomain you just created at step 1 (include `https://` in the field). e.g. `https://coolify.mydomain.com`
4. Save and wait for a minute.

You now might be able to access your Coolify panel via your custom subdomain with HTTPS.

<img width="507" alt="Screenshot 2024-03-06 at 18 21 14" src="https://github.com/coollabsio/coolify/assets/1953194/0e397fd1-e34d-455f-ba66-fa4318615bad">

**Reply by will-x86** [reply]
These appear to be self signed and give me an error when on chrome / firefox


Any ideas to fix this ?

**Reply by centralhardware** [reply]
You need to check treafic logs for any acme error and fix it so treafic can use lets' encrypt certificate

**Reply by will-x86** [reply]
I found the issue

When port 8080 is taken ( in this case by my program ) the proxy *starts* and *works* for http, but will not work for https, I am unsure why this is but when I move my programs port to 8081 it lets the proxy use port 8080 and https works
